Makefile

```diff
@@ -98,6 +98,16 @@ enter_serialize: ## run and enter serialization container for development
 test: ## run tests (set COMPILED_TAG_NAME to override default)
 	pytest tests/pytest --capture=no --verbose --refdir $(shell pwd)/tests/pytest/reference/circleci --image_version $(COMPILED_TAG_NAME)
 
+build_native: ## build FV3 locally (assuming all tools and dependencies are available in the environment)
+	$(MAKE) -j 8 -C FV3 GCOV=Y
+
+test_native: ## run native tests (all tools and build dependencies are assumed to be available in the environment)
+	find FV3 -type f -name '*.gcda' -delete
+	pytest --native tests/pytest
+	mkdir -p test_run_$(shell date -Is) && \
+		cd coverage_$(shell date -Is)  && \
+		gcovr -d -r ../FV3 --html --html-details -o index.html
```

Comment:
Something is wrong with the path name here, should it be inside `test_run_*/coverage_*` or just `coverage_*`?

Reply:
i fixed this issue....and also that the date could be different between 107 and 108.